include/libtorrent/string_view.hpp

```diff
@@ -41,11 +41,13 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/utility/string_ref.hpp>
 namespace libtorrent {
 using string_view = boost::string_ref;
+using wstring_view = boost::wstring_ref;
```

Comment:
this is really just a short-hand for the future ``std::wstring_view`` (which as far as I can tell i part of C++17 at this point). http://en.cppreference.com/w/cpp/string/basic_string_view

I don't really consider it a part of libtorrent in that sense, similar to ``error_code`` will be removed from libtorrent as soon as everyone can agree on using ``std::error_code``.